Automattic/vip-cli
# Rename scripts folder to helpers

When we deploy a new version, we deploy an extra `/scripts` folder containing two files.

`prepare-config` prepares the config we'll use to build, thus not used and shouldn't be uploaded to npm
`check-version` is used as a post installation step and will be in the `dist` folder after the build

with these change we can turn on the sensitive data check from `publish-please` back https://github.com/Automattic/vip/blob/master/.publishrc#L6

### Tests
1. Get PR
2. Run `npm run build`
3. No errors should be thrown
4. Run `npm run postinstall`
5. No errors should be thrown and no logs (file log error only if there is one)
